docs/en-US/commands/Find-JiraFilter.md

```diff
@@ -0,0 +1,298 @@
+returned.
+
+Allows any combination of the following values:
+
+- description
+- favourite
+- favouritedCount
+- jql
+- owner
+- searchUrl
+- sharePermissions
+- subscriptions
+- viewUrl
+
+```yaml
+Type: String[]
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: 4
+Default value: 'description','favourite','favouritedCount','jql','owner','searchUrl','sharePermissions','subscriptions','viewUrl'
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Sort
+
+Orders the results using one of these filter properties:
+
+- *description* Orders by filter description. Note that this ordering works independently of whether the expand to display the description field is in use.
+- *favourite_count* Orders by favouritedCount.
+- *is_favourite* Orders by favourite.
+- *id* Orders by filter id.
+- *name* Orders by filter name.
+- *owner* Orders by owner.accountId.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: 5
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -IncludeTotalCount
+
+Causes an extra output of the total count at the beginning.
+
+Note this is actually a uInt64, but with a custom string representation.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Skip
+
+Controls how many things will be skipped before starting output.
+
+Defaults to 0.
+
+```yaml
+Type: UInt64
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: 0
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -First
+
+Indicates how many items to return.
+
+```yaml
+Type: UInt64
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: 18446744073709551615
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Credential
+
+Credentials to use to connect to JIRA.
+If not specified, this function will use anonymous access.
+
+```yaml
+Type: PSCredential
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
+For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+## OUTPUTS
+
+### [JiraPS.Filter[]]
+
+## NOTES
+
+This function requires either the `-Credential` parameter to be passed or a persistent JIRA session.
+See `New-JiraSession` for more details.
+If neither are supplied, this function will run with anonymous access to JIRA.
+
+## RELATED LINKS
+
+[Get-JiraProject](../Get-JiraProject/)
```

Comment:
add `Get-JiraFilter`?

Reply:
Added both Get-JiraFilter and Get-JiraUser.